Dockerfile

```diff
@@ -57,4 +63,5 @@ COPY --from=build /myapp/build /myapp/build
 COPY --from=build /myapp/public /myapp/public
 ADD . .
 
-CMD ["npm", "start"]
+# Run with Litestream
+CMD [ "/litestream/start.sh", "npm start"]
```

Comment:
Where's this file?

Reply:
This comes from the litestream base image: https://github.com/fly-apps/litestream-base/blob/main/litestream/start.sh